luda-editor/luda-editor-client/src/app/editor/mod.rs

```diff
@@ -289,14 +314,19 @@ impl Editor {
                 }
             }
         });
+        let sequence_rc = Rc::new(sequence.clone());
         Self {
             timeline: Timeline::new(),
-            playback_time: chrono::Duration::zero(),
             image_filename_objects: vec![],
             job: None,
             clip_editor: None,
             selected_clip_id: None,
-            sequence,
+            sequence, // NOTE : I think editor should not have mutable sequence, but just immutable to track changes
```

Comment:
It's weird...
mutable sequence, but just immutable to track?
Isn't the track in the sequence?
How is this possible?

Reply:
track, not noun, for verb. -> for tracking.
I mean, sequence should be immutable so we can take a history of sequence's changes, so can implement undo-redo system (by tracking changes).